packages/kbn-openapi-common/schemas/primitives.test.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+import { NonEmptyString } from './primitives.gen';
+
+describe('NonEmptyString', () => {
+  test('accepts newline chars', () => {
+    expect(() => NonEmptyString.parse('hello \nworld')).not.toThrow();
+  });
+  test('rejects comment with just spaces', () => {
```

Comment:
Could you cover the other space characters as well like a new line symbol and tabulation? It's better to cover all possible "empty" characters like [Unicode Space Separators](https://www.compart.com/en/unicode/category/Zs) as well.

Reply:
I added the standard chars included in `\s`. I feel that should be sufficient for our use case.

Reply:
Discussed in our sync and summaried [here](https://github.com/elastic/kibana/pull/202063#issuecomment-2528354468) we decided that more tests for unicode chars would be equivalent to testing the javascript trim function. If we were continuing to use the regex it would be worthwhile to determine if the regex was parsing unicode correctly but without the regex, we should be confident trim works as expected.